spring-boot-project/spring-boot-autoconfigure/build.gradle

```diff
@@ -65,6 +65,7 @@ dependencies {
 	optional("org.codehaus.groovy:groovy-templates")
 	optional("com.github.ben-manes.caffeine:caffeine")
 	optional("com.github.mxab.thymeleaf.extras:thymeleaf-extras-data-attribute")
+	optional("com.oracle.ojdbc:ucp")
```

Comment:
Just realized I used the old groupId for Oracle. It would be better replace "com.oracle.ojdbc" with "com.oracle.database.jdbc".